cla_public/templates/base.html

```diff
@@ -113,6 +113,23 @@
 {% block javascripts %}
   <script>
 
+    $(document).ready(function(){
+      /*COVID-19: This function is a stopgap for emergency opening times*/
+      $("#field-thirdparty-time-day, #field-callback-time-day")
+        .find("option:contains(Saturday), option:contains(Sunday)")
+        .remove();
+      $("#thirdparty-time-time_in_day, #thirdparty-time-time_today, #callback-time-time_in_day, #callback-time-time_today")
+        .find("option:contains(7:30 PM), option:contains(6:30 PM), option:contains(5:30 PM)")
+        .remove();
+      var d = new Date();
+      var n = d.getDay();
+      if (n > 5) {
+        /*if we are currently in a weekend*/
+        $("#thirdparty-time-specific_day-0, #callback-time-specific_day-0").attr('disabled', 'disabled');
```

Comment:
Is it possible to just hide the "call back on weekend" option completely, so users don't need to wonder why it's disabled?

Reply:
Yes it is - I originally used `.hide()` instead, but then that leads to the question about whether we need the radio button at all as there is only one option and then that leads to awkwardness in displaying the pop-up questions.  

In the end, and in the absence of Rob, I decided to leave it as unchanged as possible.  It negates any possible layout issues and users can then see that the option is available on other days.